test/dedup-id/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "abacus-dedup-id-acceptance-tests",
+  "description": "Usage meter integration and performance tests",
+  "license": "Apache-2.0",
+  "version": "1.1.3",
+  "private": true,
+  "homepage": "https://github.com/cloudfoundry-incubator/cf-abacus/test/metering/meter",
+  "bugs": {
+    "url": "https://github.com/cloudfoundry-incubator/cf-abacus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cloudfoundry-incubator/cf-abacus.git"
+  },
+  "keywords": [
+    "cf",
+    "abacus"
+  ],
+  "files": [
+    ".npmrc",
+    "src/"
+  ],
+  "scripts": {
+    "acceptance": "abacus-eslint && abacus-mocha --command",
+    "generate-coverage": "true",
+    "lint": "abacus-eslint",
+    "pub": "publish"
+  },
+  "dependencies": {
```

Comment:
Shouldn't all of these be dev dependencies? Is there a reason for them not to be?